eventsummary/src/commonMain/kotlin/com/adammcneilly/pocketleague/eventsummary/EventSummaryListStateMutator.kt

```diff
@@ -1,29 +1,55 @@
-package com.adammcneilly.pocketleague.eventsummary.domain.state
+@file:Suppress("TooManyFunctions")
+
+package com.adammcneilly.pocketleague.eventsummary
 
 import com.adammcneilly.pocketleague.core.models.EventSummary
 import com.adammcneilly.pocketleague.core.ui.UIImage
 import com.adammcneilly.pocketleague.core.ui.UIText
-import com.adammcneilly.pocketleague.core.utils.DateTimeHelper
 import com.adammcneilly.pocketleague.event.api.GetUpcomingEventSummariesUseCase
-import com.adammcneilly.pocketleague.eventsummary.EventSummaryDisplayModel
-import com.adammcneilly.pocketleague.eventsummary.EventSummaryListViewState
-import com.tunjid.mutator.Mutation
-import com.tunjid.mutator.coroutines.stateFlowMutator
-import com.tunjid.mutator.coroutines.toMutationStream
+import com.adammcneilly.pocketleague.event.implementation.GetUpcomingEventSummariesUseCaseImpl
+import com.adammcneilly.pocketleague.event.implementation.SmashGGEventService
+import com.adammcneilly.pocketleague.eventsummary.mutator.Mutation
+import com.adammcneilly.pocketleague.eventsummary.mutator.Mutator
+import com.adammcneilly.pocketleague.eventsummary.mutator.stateFlowMutator
+import com.adammcneilly.pocketleague.eventsummary.mutator.toMutationStream
 import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.flatMapLatest
+import kotlinx.coroutines.flow.launchIn
 import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.onEach
 import kotlinx.coroutines.flow.onStart
 
+/**
+ * Helper function to create a mutator with default values so that it can be created in xcode easily.
+ */
+fun defaultMutator() = eventSummaryListStateMutator()
```

Comment:
Default arguments don't carry over from KMM to Swift, so I made this as a helper.